scripts/list_layers.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2019 Datadog, Inc.
+
+# Lists most recent layers ARNs across regions to STDOUT
+# Optionals args: [layer-name] [region]
```

Comment:
Optionals -> optional

Also, is layer-name really an arg? It looks like it's hardcoded as `dd-trace-java`